src/Compiler/Interactive/fsi.fs

```diff
@@ -1646,6 +1646,32 @@ type internal FsiDynamicCompiler(
         diagnosticsLogger.AbortOnError(fsiConsoleOutput)
         codegenResults, optEnv, fragName
 
+    /// Check FSI entries for the presence of EntryPointAttribute and issue a warning if it's found
+    let CheckEntryPoint (declaredImpls: CheckedImplFile list) =
+        let tryGetEntryPoint = function
```

Comment:
We probably can use `HaskFSharpAttribute` here, like `HasFSharpAttribute tcGlobals tcGlobals.attrib_EntryPointAttribute value.Attribs`

Reply:
Yep, that's better, thanks 👍